addons/website_forum/tests/test_controller.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+from odoo.tests.common import HttpCase, new_test_user
+
+
+class TestController(HttpCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        portal_user = new_test_user(cls.env, login='portal_user', groups='base.group_portal')
+        cls.portal = portal_user.login
+        admin_user = new_test_user(cls.env, login='admin_user', groups='base.group_user,base.group_system')
+        cls.admin = admin_user.login
+        cls.headers = {"Content-Type": "application/json"}
+        cls.pixel = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAADElEQVR4nGNgYGAAAAAEAAH2FzhVAAAAAElFTkSuQmCC'
+
+    def _build_payload(self, params=None):
+        """
+        Helper to properly build jsonrpc payload
+        """
+        return {
+            "jsonrpc": "2.0",
+            "method": "call",
+            "id": 0,
+            "params": params or {},
+        }
+
+    def test_portal_attachment_01(self):
```

Comment:
Cool to follow the [advice](https://github.com/odoo/odoo/pull/82612#discussion_r847410691) :+1: but:
```suggestion
    def test_01_portal_attachment(self):
```